tests/unit/test_query.py

```diff
@@ -756,14 +756,14 @@ def test_service_meta(search_handler, normalize_handler):
     response = search_handler.search(query)
     service_meta = response.service_meta_
     assert service_meta.name == "thera-py"
-    assert service_meta.version >= "0.2.13"
+    assert service_meta.version != "unknown"
```

Comment:
Ah, I should have done this for disease or gene? I forget which one made this check

Reply:
eh I don't really think it matters but at some point we can just run this one around everywhere